contracts/reverseRegistrar/L2ReverseClaimer.sol

```diff
@@ -0,0 +1,21 @@
+//SPDX-License-Identifier: MIT
+pragma solidity >=0.8.17 <0.9.0;
+
+import {ENS} from "../registry/ENS.sol";
+import {IL2ReverseRegistrar} from "../reverseRegistrar/IL2ReverseRegistrar.sol";
+
+contract L2ReverseClaimer {
+    constructor(
+        address l2ReverseRegistrarAddr,
+        ENS reverseRegistrar,
```

Comment:
Isn't this supposed to be registry? Why you pass this as constructor but immediately assigning an instance at line 13?